.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,55 @@
+name: 🐛 Bug
+
+description: Report an issue to help improve the project.
+
+labels: ["🛠 goal: fix"]
+
+body:
+
+  - type: textarea
+
+    id: description
+
+    attributes:
+
+      label: Description
+
+      description: A brief description of the question or issue, also include what you tried and what didn't work
+
+    validations:
+
+      required: true
+
+  - type: textarea
+
+    id: screenshots
+
+    attributes:
+
+      label: Screenshots
+
+      description: Please add screenshots if applicable
+
+    validations:
+
+      required: false
+
+  - type: textarea
+
+    id: extrainfo
+
+    attributes:
+
+      label: Additional information
+
+      description: Is there anything else we should know about this bug?
+
+    validations:
+
+      required: false
+
```

Comment:
![image](https://user-images.githubusercontent.com/34387770/209666403-ce88ee22-6bd9-40ca-9d64-a659d208f624.png)

GitHub complains that the `value` attribute must not be empty